File: src/main/java/com/prog5/items/CreditCard.java
package com.prog5.items;

import java.time.LocalDate;

public class CreditCard {
    private String owner;

    private String cardNumber;

    private LocalDate expirationDate;

    public CreditCard(String owner, String cardNumber, LocalDate expirationDate) {
        this.owner = owner;
        this.cardNumber = cardNumber;
        this.expirationDate = expirationDate;
    }

    @Override
    public String toString() {
        return "\nCreditCard: " +
                "\nowner: " + owner +
                ", cardNumber: " + cardNumber +
                ", expirationDate: " + expirationDate;
    }
}

File: CreditCard.cs
namespace Items
{
  public class CreditCard
  {
    public string Owner { get; set; }
    public string Number { get; set; }
    public string ExpirationDate { get; set; }

    public CreditCard(string owner, string number, string expirationDate)
    {
        Owner = owner;
        Number = number;
        ExpirationDate = expirationDate;
    }
  }
}

